tests/unit/cli/cli-test.js

```diff
@@ -228,6 +235,31 @@ describe('Unit: CLI', function() {
     });
   });
 
+  describe('command crash', function() {
+    it('correctly cleans up', function() {
+      let CustomCommand = Command.extend({
+        name: 'custom',
+
+        run() {
+          throw new SilentError('OMG');
+        },
+      });
+
+      project.eachAddonCommand = function(callback) {
+        callback('custom-addon', {
+          custom: CustomCommand,
+        });
+      };
+
+      return ember(['custom']).then(function() {
+        expect(false, 'should have rejected').to.eql(false);
```

Comment:
Shouldn't one of these false be a true? Seems like this line will always pass if a promise isn't rejected.